views/repo.html

```diff
@@ -55,7 +60,12 @@ <h3 class="panel-title">Deployments by Month</h3>
                 <h3 class="panel-title">URL Hash</h3>
               </div>
               <div class="panel-body">
-                <input type="text" class="form-control" readonly="readonly" value="{{url_hash}}">
+                <div class="input-group">
+                  <input type="text" class="form-control" readonly="readonly" value="{{url_hash}}" />
```

Comment:
I have too, but have started to change my tune lately. I noticed that none of the Bootstrap examples use it. Also, I think the self-closing tags are mainly a hold over from the XHTML days. Since this is not an XHTML page, the ` />` isn't needed for it to be valid markup.